contracts/test/StakeTable.t.sol

```diff
@@ -63,6 +49,22 @@ contract StakeTable_Test is Test {
             blsSig
         );
     }
+}
+
+contract StakeTable_Test is Test, StakeTableCommonTest {
```

Comment:
btw, you don't need extra `is Test`, just `is StakeTableCommonTest` is enough. 

Same applied to `Handler is CommonTest`.

Reply:
Yes, done in ced31cd.